ext/ftp/php_ftp.c

```diff
@@ -820,7 +820,7 @@ PHP_FUNCTION(ftp_nb_get)
 }
 /* }}} */
 
-/* {{{ Continues retrieving/sending a file nbronously */
+/* {{{ Continues retrieving/sending a file non-blocking */
```

Comment:
Guess the adjective is not good grammar :) Something like this maybe:
```suggestion
/* {{{ Continues to retrieve or send a file in non-blocking mode */
```

Reply:
nbronously